scripts/sanitycheck

```diff
@@ -945,6 +950,28 @@ def main():
 
         )
 
+    if (options.export_tests or options.list_tests) and options.platform:
+        if len(options.platform) > 1:
+            logger.error("When exporting tests, only platform should be specified.")
```

Comment:
"When exporting tests, only one platform should be specified." <- probably word "one" was missing